python/tk_multi_workfiles/actions/interactive_open_action.py

```diff
@@ -236,6 +236,12 @@ def _open_workfile(self, file, env, parent_ui):
                     src_path = work_path
                     work_path = local_path
 
+        try:
+            self._app.log_metric("Open workfile")
```

Comment:
For metrics whether user sandboxes are in use or not would be the interesting thing, not this level of detail (definitely not info on the dest).  I think in the original design that was there as a metric, but a low priority one.  No worries about putting it in here.  On folder creation is where I'd expect that to be gathered anyway.